integrations/clickup/clickup/client.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, AsyncGenerator
+from httpx import Timeout
+from loguru import logger
+from port_ocean.utils import http_async_client
+from enum import StrEnum
+from port_ocean.context.event import event
+
+class CacheKeys(StrEnum):
+    TEAM = "team"
+    LIST = "list" # project
+    TASK = "task" # issue
+    SPACE = "space"
+    FOLDER = "folder"
+
+class ClickupClient:
+    def __init__(self, clickup_url: str, clickup_apikey: str)-> None:
+        self.clickup_url = clickup_url
+        self.clickup_apikey = clickup_apikey
```

Comment:
You don't have to store the api key as a class variable. You can reference it directly